tests/conftest.py

```diff
@@ -43,9 +54,18 @@ def pytest_configure(config: Config) -> None:
     Configure the ethereum module and log levels to output evm trace.
     """
     if config.getoption("vmtrace", default=1) == 10:
-        config.option.__dict__["log_cli_level"] = "10"
-        config.option.__dict__["log_format"] = "%(message)s"
-        setattr(ethereum, "evm_trace", evm_trace)
+        trace.capture_op_start = capture_op_start
+        trace.capture_op_end = capture_op_end
+        trace.capture_op_exception = capture_op_exception
+        trace.capture_gas_and_refund = capture_gas_and_refund
+        trace.capture_tx_end = capture_tx_end
+        trace.output_traces = output_traces
+        trace.capture_evm_stop = capture_evm_stop
+        trace.capture_precompile_start = capture_precompile_start
+        trace.capture_precompile_end = capture_precompile_end
+        # config.option.__dict__["log_cli_level"] = "10"
+        # config.option.__dict__["log_format"] = "%(message)s"
+        # setattr(ethereum, "evm_trace", evm_trace)
```

Comment:
Are these debugging comments?

Reply:
I have removed the tracing capability from pytest and maintained it exclusively within the `t8n` tool. That gives us finer control over the tracing within t8n, for example control the output directories and file names without unwieldy workarounds. This is relevant to Dan's comments below regarding the file names.